### PR TITLE
feat: standard deployments on celo

### PR DESCRIPTION
**Pull request by yomarion**

### CODE REVIEW

File: packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts [hunk 1]
```diff
@@ -102,6 +102,15 @@ export const erc20FeeProxyArtifact = new ContractArtifact<ERC20FeeProxy>(
         },
       },
     },
+    '0.2.0-next': {
```
Comment (benjlevesque) [inline packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts:105]: what is this?

**Review by benjlevesque — APPROVED** [review]

File: packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts (hunk 1 above)
Comment (yomarion) [reply packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts:105]: I did not want to change the version just for that but I also wanted to document the new address. Do you think it's a bad idea?

Comment (benjlevesque) [reply packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts:105]: hum maybe just add a comment...

Comment (yomarion) [reply packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts:105]: You mean instead of the entry or to explain the entry?

Comment (benjlevesque) [reply packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts:105]: both :D

**Pull request merged**